modules/ppcp-button/resources/js/modules/Helper/PayPalScriptLoading.js

```diff
@@ -0,0 +1,101 @@
+import { loadScript } from '@paypal/paypal-js';
+import dataClientIdAttributeHandler from '../DataClientIdAttributeHandler';
+import widgetBuilder from '../Renderer/WidgetBuilder';
+import { processConfig } from './ConfigProcessor';
+
+const loadedScripts = new Map();
+const scriptPromises = new Map();
```

Comment:
I think this will not be fully global. We import this file in other modules and then webpack builds separate JS assets. So maybe e.g. putting it into `window.` would be more correct. But not sure if matters currently.